features/support/paths.rb

```diff
@@ -10,6 +10,8 @@ def path_to(page_name)
 
     when /the home\s?page/
       '/'
+    when /the media page/
```

Comment:
It's actually because there's a tricky issue with two of the scenarios in `features/other_a/media.feature`. If you omit this, those scenarios run into an error when calling `path_to_pickle` to get the media page:
```
Could not figure out a path for ["the media"] {} (RuntimeError)
./features/support/paths.rb:37:in `path_to'
./features/step_definitions/web_steps.rb:45:in `/^(?:|I )go to (.+)$/'
[...]/.rvm/rubies/ruby-2.6.5/bin/bundle:23:in `load'
[...]/.rvm/rubies/ruby-2.6.5/bin/bundle:23:in `<main>'
features/other_a/media.feature:42:in `When I go to the media page'
```
It's related to introducing the `:media` factory and removing the explicit `a media exists with name: "NAME", canonical: true` step (which conflicts with the step introduced by adding the factory), but I've never figured it out beyond that.

(Oddly enough, only the scenarios that *don't* use that step that seem to trigger the error. I considered adding an extra `a media exists` step to the two buggy scenarios instead of adding a new path case, but that wouldn't help anyone who wants to add future tests involving the media page.)